assets/scripts/pages/search.js

```diff
@@ -79,14 +79,21 @@ window.addEventListener('DOMContentLoaded', () => {
       // pull template from hugo template definition
       const templateDefinition = document.getElementById('search-result-template').innerHTML
       // replace values
+      function adaptTags() {
+        const tags = value.item.tags;
```

Comment:
You need to check if tags null or not. The search on deploy preview fails with this error:
```bash
application.e4d15ee8d45ef1141f3853aff6a4dcb0489644435baa1e71cf89f5e1c292ade8.js:279 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'forEach')
    at y (application.e4d15ee8d45ef1141f3853aff6a4dcb0489644435baa1e71cf89f5e1c292ade8.js:279:32904)
    at application.e4d15ee8d45ef1141f3853aff6a4dcb0489644435baa1e71cf89f5e1c292ade8.js:279:33145
    at Array.forEach (<anonymous>)
    at n (application.e4d15ee8d45ef1141f3853aff6a4dcb0489644435baa1e71cf89f5e1c292ade8.js:279:32361)
    at application.e4d15ee8d45ef1141f3853aff6a4dcb0489644435baa1e71cf89f5e1c292ade8.js:279:32212
```